integreat_cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -22,6 +22,14 @@ msgstr ""
 msgid "API"
 msgstr "API"
 
+#: api/v3/socialmedia_headers.py
+msgid ""
+"hallo aschaffenburg is your digital companion for the town of Aschaffenburg. "
+"Here you can find local information, advice centres and services."
+msgstr ""
+"hallo aschaffenburg is your digital companion for the town of Aschaffenburg. "
+"Here you can find local information, advice centres and services."
```

Comment:
```suggestion
"hallo aschaffenburg ist Ihr digitaler Begleiter für die Stadt Aschaffenburg. "
"Hier finden Sie lokale Informationen, Beratungsstellen und Dienstleistungen."
```

Reply:
I intentionally added the English translation here. As we do not have language tags here, we probably want to only use the English translations (as of now). But maybe I can fully remove the translation.

Reply:
Ah ok, but then I'd use the
```
from django.utils import translation

with translation.override("en"):
    excerpt = _("...")
```
or wouldn't use the gettext-function at all, since this way we might forget about it and just have missings translations in our po file that we don't know about...

Reply:
yes, lets fully remove the translation.